instrumentation/akka-http-core-2.13_10.1.8/src/main/scala/akka/http/scaladsl/SyncRequestHandler.scala

```diff
@@ -0,0 +1,61 @@
+/*
+ *
+ *  * Copyright 2020 New Relic Corporation. All rights reserved.
+ *  * SPDX-License-Identifier: Apache-2.0
+ *
+ */
+
+package akka.http.scaladsl
+
+import java.util.logging.Level
+
+import akka.http.scaladsl.model.{HttpRequest, HttpResponse}
+import com.newrelic.agent.bridge.AgentBridge
+import com.newrelic.api.agent.weaver.Weaver
+import com.newrelic.api.agent.{NewRelic, Trace, Transaction, TransactionNamePriority}
+import com.nr.instrumentation.akkahttpcore.{RequestWrapper, ResponseWrapper}
+
+import scala.runtime.AbstractFunction1
+
+class SyncRequestHandler(handler: HttpRequest => HttpResponse) extends AbstractFunction1[HttpRequest, HttpResponse] {
+
+  val transactionCategory :String = "AkkaHttpCore"
+
+  @Trace(dispatcher = true)
+  override def apply(param: HttpRequest): HttpResponse = {
+
+    try {
+      NewRelic.getAgent.getTransaction.setTransactionName(TransactionNamePriority.FRAMEWORK_LOW, true, transactionCategory, "akkaHandler")
```

Comment:
Is there a reason you used a different TransactionNamePriority here?

Reply:
That was inadvertent.  I initially tried to update the instrumentation to avoid the deprecated new relic features, but ended up getting a bit stuck and opted to use the same style as the 2.11 version of the instrumentation.  I will fix this.

Reply:
@tspring I changed this back to the SERVLET_NAME priority.